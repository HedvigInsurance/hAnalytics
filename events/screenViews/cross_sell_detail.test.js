const validateEvent = require("../../commons/validateEvent.js")

test('validate cross_sell_detail', () => {
    const event = validateEvent(__filename)
    expect(event).toMatchSnapshot()
});