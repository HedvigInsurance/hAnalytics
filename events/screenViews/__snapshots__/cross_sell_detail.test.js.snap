// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate cross_sell_detail 1`] = `
Object {
  "name": "screen_view_cross_sell_detail",
  "variables": Object {
    "type_of_contract": undefined,
  },
}
`;
